mu4/scenes/inspector/internal/compatibility/mu3inspectoradapter.cpp

```diff
@@ -0,0 +1,192 @@
+#include "mu3inspectoradapter.h"
```

Comment:
It is better to put this adapter in `mscore`, otherwise it causes dependence on `mscore`